src/bandersnatch/mirror.py

```diff
@@ -290,6 +294,52 @@ def record_finished_package(self, name: str) -> None:
                 ]
                 f.write("\n".join(todo))
 
+    async def cleanup_non_pep_503_paths(self, package: Package) -> None:
+        """
+        Before 4.0 we use to store backwards compatible named dirs for older pip
+        This function checks for them and cleans them up
+        """
+
+        def raw_simple_directory() -> Path:
```

Comment:
These are only used for cleanup so I stuck them inline

Reply:
FWIW - these use to be used else where but I clean all that up.